libr/socket/rap_server.c

```diff
@@ -17,6 +30,12 @@ R_API RSocketRapServer *r_socket_rap_server_new (int is_ssl, const char *port) {
 	return NULL;
 }
 
+R_API RSocketRapServer *r_socket_rap_server_write(RSocketRapServer *rap_s, char *cmd){
+	strcpy(rap_s->buf, cmd);
```

Comment:
imho RSocketRapServer should be using ut8 *buf instead of `ut8 buf[4101]`. mainly because that constant thing is ugly (pun in @condret) and there's no reason to make this variable that big, so we can just strdup instead of strcpy for example